### PR TITLE
Andy hee patch 2

### PR DESCRIPTION
**Pull request by AndyHee**
I rephrased this, as it was rather convoluted and contained culturally specific instructions that may not be generally applicable.

I hope these two extra strings add not too much burden on the translator teams.

(Sorry about the mess about message.po. I had forgotten that this is automatically generated.)

### CODE REVIEW

File: nsfw/nsfw.php [hunk 1]
```diff
@@ -80,11 +80,11 @@ function nsfw_addon_settings(&$a,&$s) {
     $s .= '</span>';
     $s .= '<div id="settings_nsfw_expanded" class="settings-block" style="display: none;">';
     $s .= '<span class="fakelink" onclick="openClose(\'settings_nsfw_expanded\'); openClose(\'settings_nsfw_inflated\');">';
-    $s .= '<h3>' . L10n::t('Not Safe For Work (General Purpose Content Filter)') . '</h3>';
+    $s .= '<h3>' . L10n::t('Content Filter(#NSFW tag and more)') . '</h3>';
```
Comment (MrPetovan) [inline nsfw/nsfw.php]: Please add a space before the opening parenthesis.

I wouldn't put the hashtag in the name as well, just `Content Filter (NSFW and more)`

**Comment by MrPetovan** [comment]
Apparently you have a conflict with the base `3.6-rc` branch, are you sure you created your branch directly from it?

**Comment by AndyHee** [comment]
Thanks! Yes, I fix this space and has tag issue.

Let me find out what's going on here with my branch.

**Comment by MrPetovan** [comment]
If you're on Windows or Mac OS X, you can use the Git Desktop app that simplifies the git workflow, you can create branches from other branches and avoid conflicts.

**Comment by AndyHee** [comment]
I'm on Linux. I've checked out the core branch and anything worked OK with git via command line.

Now I've tried to use the web interface and created this mess. I'll check out the addon branch properly and do it the normal way.

**Pull request closed without merging**